VirtoCommerce.Storefront/Controllers/StaticContentController.cs

```diff
@@ -88,6 +88,34 @@ public async Task<ActionResult> GetContentPageByName(string page)
                     return Challenge();
                 }
                 SetCurrentPage(contentPage);
+                if (contentPage.FileName.EndsWith(".json"))
```

Comment:
This code can be removed for case of use the existing static content types (ContentPage and BlogArticle) for designer content presentation

Reply:
removed